### PR TITLE
unused local variables removed

### LINKED ISSUE #710: fix warnings generated from lgtm
https://lgtm.com/projects/g/aau-network-security/haaukins?mode=list

### PR DESCRIPTION
**Pull request by mrtrkmn**
Fixes #710

### CODE REVIEW

File: store/exercise.go [hunk 1]
```diff
@@ -170,9 +170,8 @@ func (ic InstanceConfig) Validate() error {
 	if ic.Image == "" {
 		return &EmptyVarErr{Var: "Image", Type: "Instance ServiceConfig"}
 	}
-	if ic.MemoryMB < 0 {
-		return errors.New("memory cannot be negative")
-	}
+	// memoryMB is UINT, UINT cannot be negative,
+	// in case memoryMB is empty, it will be automatically handled
```
Comment (Lanestolen) [inline store/exercise.go:174]: Remove these comments ;)

File: main.go [hunk 2]
```diff
@@ -62,6 +62,10 @@ func main() {
 
 	// ensure that gRPC port is free to allocate
 	conn, err := net.DialTimeout("tcp", daemon.MngtPort, time.Second)
+	if err != nil {
+		fmt.Printf("Error on DialTimeout %v\n", err)
```
Comment (Lanestolen) [inline main.go:66]: Maybe make this as a log.Fatal().Err(err).....

File: lab/hub.go [hunk 3]
```diff
@@ -179,7 +179,7 @@ func (h *hub) Suspend(ctx context.Context) error {
 		wg.Add(1)
 		go func() {
 			if err := l.Suspend(ctx); err != nil {
-				err = suspendError
+				suspendError = err
```
Comment (Lanestolen) [inline lab/hub.go:182]: As far as i know these errors is not checked maybe just a log statement

**Review by Lanestolen — APPROVED** [review]
Three minor comments,  when implemented feel free to merge

File: lab/hub.go (hunk 3 above)
Comment (mrtrkmn) [reply lab/hub.go:182]: hmm those are checked in functions which are calling Suspend function actually. 
for instance here: https://github.com/aau-network-security/haaukins/blob/4b7f5c13ff8ab5cb7ef2ac1b169d57855836a5c9/svcs/guacamole/event.go#L587

Comment (mrtrkmn) [reply lab/hub.go:182]: I actually realized that this is useless, I will remove it as well 
https://github.com/aau-network-security/haaukins/blob/4b7f5c13ff8ab5cb7ef2ac1b169d57855836a5c9/svcs/guacamole/event.go#L586

Comment (Lanestolen) [reply lab/hub.go:182]: Then it is perfect :ok_hand:

File: main.go [hunk 4]
```diff
@@ -63,24 +64,24 @@ func main() {
...
 		_ = conn.Close()
-		fmt.Printf("Checking gRPC port %s report: %v\n", daemon.MngtPort, daemon.PortIsAllocatedError)
+		log.Fatal().Err(errors.New(fmt.Sprintf("Checking gRPC port %s report: %v\n", daemon.MngtPort, daemon.PortIsAllocatedError)))
 		return
 	}
 
 	c, err := daemon.NewConfigFromFile(*confFilePtr)
 	if err != nil {
-		fmt.Printf("unable to read configuration file \"%s\": %s\n", *confFilePtr, err)
+		log.Fatal().Err(errors.New(fmt.Sprintf("unable to read configuration file \"%s\": %s\n", *confFilePtr, err)))
```
Comment (Lanestolen) [inline main.go:78]: I think you'll need a .msg for it to be printed :)

**Comment by mrtrkmn** [comment]
it would be nice to take this value from configuration file instead of binding it to binary:
https://github.com/aau-network-security/haaukins/blob/c4e1f948bb496efaa9cfca1f517b104a6ffec2e6/daemon/daemon.go#L58

I am going to create issue related to it, if you agree. @Lanestolen

**Pull request merged**